### PR TITLE
[travis] Add timing info

### PR DESCRIPTION
**Pull request by nicolas-grekas**
| Q             | A
| ------------- | ---
| Branch?       | 2.7
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | -
| License       | MIT
| Doc PR        | -

Let's see if this works.
For reference, see https://github.com/travis-ci/travis-build/blob/master/lib/travis/build/templates/header.sh

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -51,13 +51,32 @@ before_install:
       export PHPUNIT_X="$PHPUNIT --exclude-group tty,benchmark,intl-data"
       export COMPOSER_UP='composer update --no-progress --no-suggest --ansi'
 
+      nanoseconds() {
+          local cmd="date"
+          local format="+%s%N"
+          local os=$(uname)
+          if hash gdate > /dev/null 2>&1; then
+            cmd="gdate"
+          elif [[ "$os" = Darwin ]]; then
```
Comment (sstok) [inline .travis.yml]: We don't test on macOS (yet) right?

**Comment by stof** [comment]
the output is weird. the timing gets on a next line. This may not be the right ordering.

File: .travis.yml (hunk 1 above)
Comment (nicolas-grekas) [inline .travis.yml]: right, I just borrowed this code from elsewhere

**Comment by nicolas-grekas** [comment]
Works great!

capture du 2017-08-24 13-18-09

**Pull request merged**